app/main.py

```diff
@@ -1,3 +1,53 @@
+from __future__ import annotations
+
+
 class Distance:
     # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other_distance: int | float | Distance) -> Distance:
+        if isinstance(other_distance, (int, float)):
+            return Distance(self.km + other_distance)
+        return Distance(self.km + other_distance.km)
+
+    def __iadd__(self, other_distance: int | float | Distance) -> Distance:
+        if isinstance(other_distance, (int, float)):
+            self.km += other_distance
+            return self
+        self.km += other_distance.km
+        return self
+
+    def __mul__(self, number: int | float) -> Distance:
+        if isinstance(number, (int, float)):
+            return Distance(self.km * number)
+
+    def __truediv__(self, number: int | float) -> Distance:
+        return Distance(round(self.km / number, 2))
```

Comment:
why does u make check in mul method, but don't check for type in truediv method ?